pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -182,7 +194,9 @@ func (rc *reconciler) reconcile() {
 			}
 		}
 	}
+}
 
+func (rc *reconciler) mountAttachVolumes() {
 	// Ensure volumes that should be attached/mounted are attached/mounted.
 	for _, volumeToMount := range rc.desiredStateOfWorld.GetVolumesToMount() {
 		volMounted, devicePath, err := rc.actualStateOfWorld.PodExistsInVolume(volumeToMount.PodName, volumeToMount.VolumeName)
```

Comment:
PodExistsInVolume() returns true for MountUncertain volumes too?

Reply:
good catch! fixed and added test.